packages/mesa/build.sh

```diff
@@ -51,12 +53,21 @@ termux_step_post_configure() {
 	rm -f $_WRAPPER_BIN/cmake
 }
 
-termux_step_post_massage() {
+termux_step_post_make_install() {
 	# A bunch of programs in the wild assume that the name of OpenGL shared
 	# library is `libGL.so.1` and try to dlopen(3) it. In fact `sdl2` does
 	# this. So please do not ever remove the symlink.
-	cd ${TERMUX_PKG_MASSAGEDIR}/${TERMUX_PREFIX}/lib || exit 1
-	if [ ! -e "./libGL.so.1" ]; then
-		ln -sf libGL.so libGL.so.1
+	if [ ! -e "${TERMUX_PREFIX}/lib/libGL.so.1" ]; then
+		ln -sf libGL.so ${TERMUX_PREFIX}/lib/libGL.so.1
 	fi
+	if [ ! -e "${TERMUX_PREFIX}/lib/libEGL.so.1" ]; then
+		ln -sf libEGL.so ${TERMUX_PREFIX}/lib/libEGL.so.1
+	fi
+	if [ ! -e "${TERMUX_PREFIX}/lib/libGLESv1_CM.so.1" ]; then
+		ln -sf libGLESv1_CM.so ${TERMUX_PREFIX}/lib/libGLESv1_CM.so.1
+	fi
+	if [ ! -e "${TERMUX_PREFIX}/lib/libGLESv2.so.2" ]; then
+		ln -sf libGLESv2.so ${TERMUX_PREFIX}/lib/libGLESv2.so.2
+	fi
```

Comment:
Always make symlinks if you do that in `termux_step_post_make_install`. Otherwise they won't be included in the bindist package when already exist.